agent/ecscni/plugin.go

```diff
@@ -180,11 +187,18 @@ func (client *cniClient) cleanupNS(cfg *Config) error {
 		return errors.Wrap(err, "cni cleanup: invoke bridge plugin failed")
 	}
 	seelog.Debugf("[ECSCNI] bridge cleanup done: %s", cfg.ContainerID)
-
+	// clean up eni network namespace
 	err = client.del(runtimeConfig, cfg, client.createENINetworkConfig)
 	if err != nil {
 		return errors.Wrap(err, "cni cleanup: invoke eni plugin failed")
 	}
+	if cfg.AppMeshCNIEnabled {
```

Comment:
Is it possible that cfg is nil when u call cfg.AppMeshCNIEnabled here?

Reply:
discussed offline, cfg is always constructed or will be error out before here.